src/controller/GuideConroller.ts

```diff
@@ -0,0 +1,32 @@
+import {
+  Controller,
+  Get,
+  Path,
+  Route,
+} from "tsoa";
+import { GuideService } from "../service/GuideService";
+import StatusCode from "../utils/StatusCode";
+import { errorMessage } from "../utils/errorMessage";
+
+@Route("guide")
+export class GuideRouter extends Controller {
+
+ @Get("{guideId}")
+ public async findGuide( @Path() guideId: string ) {
+
+  const data = await new GuideService().findGuide(guideId);
+  
+  if (data === null) {
+    
+    this.setStatus(400)
+    
+    return {
+      status: StatusCode.BAD_REQUEST,
+      messsage : errorMessage.BAD_REQUEST
+    }
```

Comment:
```suggestion
  if (data === null) {
    this.setStatus(400)
    const notFoundGuide =  {
      status: StatusCode.BAD_REQUEST,
      message: errorMessage.BAD_REQUEST
    }
    return notFoundGuide
```
바로 return 해도 상관 없지만 만약 에러가 여러가지라면 각 에러를 담은 변수로 가독성을 올릴 수 있을 것 같다는 생각이 듭니다.

위에서 data 라는 변수 또한 그렇지만 클라이언트에 보내는 데이터는 실제 있는 데이터일 수도 있고 그렇지 않은 에러에 대한 데이터일 수 있으니 이 경우는 네이밍을 정하여 명확하게 처리하는 것이 추후 유지보수 측면에서 더  좋지 않을 까요?

Reply:
넵 참고해서 수정하겠습니다!!